examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
+    const obj = instance || {};
+
+    // Set up prototype
+    Object.assign(proto, descriptor.methods);
+    Object.setPrototypeOf(obj, proto);
```

Comment:
We should avoid making changes to object's [[Prototype]]. Instead we can just create a new object.

```js
const proto = {};
let obj = instance || {};

// Set up prototype
Object.assign(proto, descriptor.methods);

if (isEmpty(Object.getPrototypeOf(obj))) {
  obj = Object.create(proto);
}
```